newapi/ooniapi/probe_services.py

```diff
@@ -460,12 +460,22 @@ def list_test_helpers():
             },
         ],
     }
-    return cachedjson(1, **j)
+    probe_ipaddr = request.headers.get("X-Real-Ip", "")
+    if probe_ipaddr.startswith("2.34."):
+        # Temporary hack to match 2.34.0.0/16 announced by AS30722
+        j["web-connectivity"] = [
+            {"address": "https://0.th.ooni.org", "type": "https"},
+            {
+                "address": "https://d33d1gs9kpq1c5.cloudfront.net",
+                "front": "d33d1gs9kpq1c5.cloudfront.net",
+                "type": "cloudfront",
+            },
+        ]
+    return cachedjson(0, **j)
```

Comment:
For the records: this line change (`1` => `0`) means we're disabling caching.

Of course, this diff would not work with caching enabled.